src/setup-docker/templates/fabric-docker/scripts/chaincode-functions-v2.sh

```diff
@@ -120,8 +120,10 @@ chaincodePackageCCaaS() {
   inputLog "CONTAINER_PORT: $CONTAINER_PORT"
   inputLog "TLS_ENABLED: $TLS_ENABLED"
 
-  # Use the same container name logic as startCCaaSContainer
+  
+  # local CHANNEL_NAME=$9  
```

Comment:
please verify a Fablo config with two channels, each one with CCaaS chaincode, with the same name. I believe it fails in this case